File: new/atoms/tour-stop-label.test.mjs
import * as m from './tour-stop-label.mjs';
const SNAP = {"TOUR_STOP_LABEL":"\"■ עצירת הדמיה (Esc)\""};
let f = 0;
for (const [k, s] of Object.entries(SNAP)) if (JSON.stringify(m[k]) !== s) { console.error('✗ ' + k + ' סטה מהצילום'); f = 1; }
if (f) process.exit(1); console.log('✓ tour-stop-label: צילום-ערך תואם — ירוק');

File: new/dart-maor/tour-stop-label_test.dart
// בדיקת-חוזה (רתמת-זהב) · tour-stop-label — צילום-ערך jsonEncode≡JSON.stringify.
// חוללה מכנית (gen-data-dart). הרצה: dart run --enable-asserts ⇒ OK
import 'dart:convert';
import 'tour-stop-label.dart';

void main() {
  // ‏1) צילום-ערך: jsonEncode ≡ JSON.stringify של המקור (זהה-ביט).
  if (jsonEncode(tourStopLabel) != '"■ עצירת הדמיה (Esc)"') {
    throw StateError('FAIL tourStopLabel: צילום-הערך סטה');
  }
  print('OK tour-stop-label: 1 data-snapshot(s) passed');
}
